doc/rippled-example.cfg

```diff
@@ -824,6 +824,10 @@
 #                           require administrative RPC call "can_delete"
 #                           to enable online deletion of ledger records.
 #
+#       genesis_seq         The earliest ledger sequence to start from. The
+#                           ripple ledger consensus network default is
+#                           32570, alternate networks may set this value.
```

Comment:
Consider rephrasing to make it clear is 32570 is the application default. As written, the reader might believe 32570 is only the default when connecting to the main network. Perhaps:
```
The default is 32570 to match the XRP ledger network's earliest allowed sequence. Alternate networks may set this value.
```

Reply:
Thanks. 👍